platform/broadcom/docker-syncd-brcm/Dockerfile.j2

```diff
@@ -30,6 +30,24 @@ COPY ["supervisord.conf", "/etc/supervisor/conf.d/"]
 COPY ["files/supervisor-proc-exit-listener", "/usr/bin"]
 COPY ["critical_processes", "/etc/supervisor/"]
 
+# Install make and git for compiling librdb
+RUN apt-get -y install build-essential git
+
+# Obtain librdb source codes
+RUN git clone https://github.com/redis/librdb.git
+
+# Build librdb
+RUN cd ./librdb/;git checkout b401df1e7025501c24f0c852aa32f6900fd888f7;git submodule update --init --recursive;make;cd ..
```

Comment:
Instead of building here in a Dockerfile, please build it just like any other regular application.

Reply:
Hi @saiarcot895 , please give me some hints, such as one regular application for reference. Thanks.

Reply:
See https://github.com/sonic-net/sonic-buildimage/blob/master/rules/ixgbe.mk and https://github.com/sonic-net/sonic-buildimage/blob/master/src/ixgbe/Makefile as a basis. In this case, you'd need to do a git clone of the repo (instead of wget), build `librdb`, and then copy the static binary to `$(DEST)`.

I looked at their repo and build step, and I'm not a fan of using a fixed version of the hiredis submodule rather than using `hiredis.h` that may be installed system-wide, but for now, that's fine.

Reply:
You'd also need to update https://github.com/sonic-net/sonic-buildimage/blob/master/platform/broadcom/docker-syncd-brcm.mk to add this into the `$(DOCKER_SYNCD_BASE)_FILES` list.

Reply:
@saiarcot895 ,  @mlok-nokia , please help to review and approve. Thanks.

Reply:
Hi @saiarcot895 , the checking item below blocked the progress. How can I deal with it?
_Azure.sonic-buildimage Expected — Waiting for status to be reported_

Reply:
Can you make a comment on the PR (not within this issue thread) with `/azpw run`?

Reply:
Thanks.

Reply:
Hi @deepak-singhal0408 , @saiarcot895 , there are still three checking errors, how to fix it? Thanks. <img width="1264" alt="image" src="https://github.com/user-attachments/assets/2c92a66f-3f58-4538-80e1-9c873d131701">